client/apps/edit/components/content/sections/embed/index.coffee

```diff
@@ -1,124 +1,42 @@
-            a {
-              name: 'overflow_fillwidth'
-              className: 'layout'
-              onClick: @changeLayout('overflow')
-              'data-active': @props.section.get('layout') is 'overflow'
-            }
-            a {
-              name: 'column_width'
-              className: 'layout'
-              onClick: @changeLayout('column_width')
-              'data-active': @props.section.get('layout') is 'column_width'
-            }
-          section { className: 'ese-inputs' },
-            div {
-              className: 'ese-iframe-error'
-              dangerouslySetInnerHTML: __html: @state.errorMessage
-            }
-            form {
-              className: 'ese-input-form-container'
-              onSubmit: @submitUrl
-            },
-              div { className: 'ese-input' }, "URL",
-                input {
-                  placeholder: 'https://files.artsy.net'
-                  className: 'bordered-input bordered-input-dark'
-                  ref: 'url'
-                  defaultValue: @props.section.get('url')
-                },
-              div { className: 'ese-input ese-input-height' }, "Height (optional)",
-                input {
-                  placeholder: '400'
-                  className: 'bordered-input bordered-input-dark'
-                  ref: 'height'
-                  defaultValue: @props.section.get('height')
-                }
-              div { className: 'ese-input ese-input-height' }, "Mobile Height (optional)",
-                input {
-                  placeholder: '300'
-                  className: 'bordered-input bordered-input-dark'
-                  ref: 'mobileHeight'
-                  defaultValue: @props.section.get('mobile_height')
-              }
-              button {
-                className: 'avant-garde-button avant-garde-button-dark'
-                'data-state': if @state.loading then 'loading' else ''
-              }, 'Add URL'
-      div {
-        className: 'embed-container'
-        ref: 'embed'
-      }
-        (if @state.url is ''
-          div { className: 'ese-empty-placeholder' }, 'Add URL above'
-        else
-          div {
-            className: 'ese-embed-content'
-          },
-            iframe {
-              src: @state.url
-              className: 'embed-iframe'
-              scrolling: 'no'
-              frameBorder: '0'
-              style: { 'height': @state.height if @state.height?.length }
-            }
-        )
+          article: @props.article
+        }
+      if !@props.section.get('url')?.length
+        div { className: 'edit-section--embed__placeholder' }, 'Add URL above'
+      else
+        div { className: 'edit-section--embed__content' },
+          Embed {
+            section:
+              url: @props.section.get 'url'
+              height: @props.section.get 'height'
+              mobile_height: @props.section.get 'mobile_height'
+              layout: @props.section.get 'layout'
```

Comment:
Backbone gives you a nice `toJSON` method that will let you avoid this. This with a combo of one of the underscore/lodash methods could help clean this up a bit!